packages/radix-ui-themes/src/components/radio-card-group.css

```diff
@@ -6,15 +6,18 @@
 .rt-RadioCardGroupItem {
   display: flex;
   align-items: center;
-  white-space: nowrap;
   min-width: 0;
 }
 
-.rt-RadioCardGroupItem > * {
-  /* Avoid unintentional drag interactions (e.g. on images) */
-  pointer-events: none;
-  /* Don’t squish icons */
-  flex-shrink: 0;
+.rt-RadioCardGroupItemInner {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+
+  & * {
+    /* Avoid unintentional drag interactions (e.g. on images) */
+    pointer-events: none;
+  }
```

Comment:
- Is there any way we could avoid having an inner element? This would make passing a `style` prop much more viable, especially when overriding the default layout.
  - I scouted the internet for Radio Card examples in the wild and truncation by default is probably not such a great idea, e.g. [Apple](https://www.apple.com/shop/buy-mac/mac-pro/rack). Should we leave it to consumers to compose with `<Text truncate>`? (Just need to make sure it works, e.g. via `& > * { min-width: 0 }` on the items)
- There's a regression on the layout with icons:
  - https://themes-playground-git-checkbox-card-group-modulz.vercel.app/test-radio-card
  - https://themes-playground-modulz.vercel.app/test-radio-card
- Turbo nit: it was `& > *` and not `& *` so that `pointer-events: none` doesn't pollute the styles of the inner elements for a nicer web inspector DX

Reply:
Also thinking that we should default to justifying the content to center by default as a more vanilla option

Reply:
Yeah happy to drop the default truncation inside the component itself. 

> Should we leave it to consumers to compose with <Text truncate>

I also agree with this, that's how it works already when users have their own custom layout inside.

> There's a regression on the layout with icons

That's not a regression, I actually removed the default layout accommodating for icons, etc as I thought we'd say we perhaps shouldn't do it and let users lay things out outside of a trivial text inside the card? I just forgot to update this page to remove the icons.

But let me know if you think we should actually keep the convenience of flex + gap for text/icons by default.